connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala

```diff
@@ -133,11 +136,17 @@ class SparkConnectClientSuite extends ConnectFunSuite with BeforeAndAfterEach {
     TestPackURI(
       "sc://localhost:1234/",
       isCorrect = true,
-      client => testClientConnection(1234)(_ => client)),
+      client => {
```

Comment:
@vicennial I removed the code to create a real connection as we've got it covered by

https://github.com/apache/spark/blob/cd6aa5819f3e97614532ec1f8c8d03b2a78c9340/connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala#L91-L95

Reply:
Can we also add a port-less connection string test?

Reply:
LGTM otherwise!

Reply:
I kept the port-less one to still start the server as it is kind of important use case. So all good! See a few lines above the test.

https://github.com/apache/spark/blob/cd6aa5819f3e97614532ec1f8c8d03b2a78c9340/connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/connect/client/SparkConnectClientSuite.scala#L129-L132